.github/workflows/test.yml

```diff
@@ -41,7 +41,10 @@ jobs:
         pip install .[ml]
     - name: Test with pytest
       run: |
-        python -m pytest --random-order --random-order-bucket=global --ignore=docs -m "not api_validation" tests/
+        # unit-tests
+        python -m pytest --random-order --random-order-bucket=global --ignore=docs --ignore=tests/test_app.py tests/
+        # application test
+        python -m pytest --random-order --random-order-bucket=global --ignore=docs tests/test_app.py
```

Comment:
I assume that new test command can be separated to a new step in the `test.yml`

Reply:
Unit tests might be used for code coverage. Separating steps lead to overhead in workflow - double python setup.

Reply:
I meant something like

```yaml
- name: Unit tests
      run: ...
- name: Application tests
      run: ... 
```

That should not add any overhead, just additional separation between different types of tests 

![image](https://user-images.githubusercontent.com/43581724/157858049-a16de593-04ae-47f3-ae21-a0fa5439164f.png)

Reply:
ok